src/app/components/SubjectHeading/BibsList.jsx

```diff
@@ -204,13 +205,14 @@ class BibsList extends React.Component {
     }
     const bibResults = bibsSource === 'discoveryApi' ? results : results.slice(this.firstBib(), this.lastBib());
 
-    const h3Text = `Viewing ${this.firstBib() + 1} - ${this.lastBib()} of ${totalResults || ''} items for Heading "${label}"`;
+    const h3Text = `Viewing ${this.firstBib() + 1} - ${this.lastBib()} of ${totalResults || ''} item${(totalResults || 0) > 1 ? 's' : ''}`;
```

Comment:
is it possible `totalResults` could be `0` here leading to something like "Viewing 0 - 0 of item" ?

Reply:
Good point. I think these default values are more intended to handle the case where `totalResults` is not defined for some reason. Since the message doesn't really make any sense in that case, I've changed it to only display the second half of the message if `totalResults` is a number, and to only show the singular case if `totalResults === 1`